.vscode/settings.json

```diff
@@ -84,7 +84,12 @@
         "__split_buffer": "cpp",
         "filesystem": "cpp",
         "queue": "cpp",
-        "stack": "cpp"
+        "stack": "cpp",
+        "__errc": "cpp",
+        "__hash_table": "cpp",
+        "__string": "cpp",
+        "algorithm": "cpp",
+        "codecvt": "cpp"
```

Comment:
You really should put a `"**/include/c++/**": "cpp",` line in here so VSCode stops spamming every libc++ header here.

Reply:
Also, ewww `<codecvt>`.